src/Routes/RosPage/RosPage.js

```diff
@@ -465,8 +464,8 @@ class RosPage extends React.Component {
         return (
             <React.Fragment>
                 <PermissionContext.Consumer>
-                    { value =>
-                        value.permissions.systemsRead === false
+                    { hasPermissions =>
+                        hasPermissions === false
```

Comment:
```suggestion
                    { value =>                        
                        value.permissions.hasRead === false
```

Instead of modifying whole structure of object, could you just rename `systemsRead` to `hasRead` ?